docs/public/static/error-codes.json

```diff
@@ -19,5 +19,6 @@
   "18": "MUI: The provided shorthand %s is invalid. The format should be `@<breakpoint | number>` or `@<breakpoint | number>/<container>`.\nFor example, `@sm` or `@600` or `@40rem/sidebar`.",
   "19": "MUI: The `experimental_sx` has been moved to `theme.unstable_sx`.For more details, see https://github.com/mui/material-ui/pull/35150.",
   "20": "MUI: `vars` is a private field used for CSS variables support.\nPlease use another name.",
-  "21": "MUI: The `colorSchemes.%s` option is either missing or invalid."
+  "21": "Invalid %s `%s` supplied to `%s`. Expected an HTMLElement.",
+  "22": "MUI: The `colorSchemes.%s` option is either missing or invalid."
```

Comment:
Is there a way we can ensure codes don't switch places like this? We should probably be consistent between versions.